src/components/modals/SharedLinkSettingsModal/SharedLinkSettingsModal.tsx

```diff
@@ -138,12 +138,12 @@ export const SharedLinkSettingsModal: React.FC<SharedLinkSettingsModalProps> = (
 
     // If we already have a generated share link, copy it
     if (generatedShareLink) {
-      Clipboard.setString(generatedShareLink);
+      await Clipboard.setStringAsync(generatedShareLink);
 
       return;
     }
 
-    const isFolder = item?.type ? false : true;
+    const isFolder = item?.fileId ? false : true;
```

Comment:
Should use isFolder property here, but is not common to all the interfaces we have. We have DriveItem, DriveListItem, DriveItemProps... is a bit messy and not all of them extends from the DriveItem base interface which makes this a nice refactor candidate